### DISCUSSION TITLE
Action folders

### DISCUSSION THREAD

**Discussion by smitmartijn** [elgatosf/elgato#10] [Ideas]
Hi 👋

I'm working on updating a plugin and I've added 20+ actions, and I'd really like to see action folders in the Stream Deck configuration app, as to make the user-experience easier. Most of my actions can be categorised in batches of 4-ish, and folders would make it a lot easier for the end user. Folder names can also add more context around the action which would be helpful to understand what it does.

There are several other plugins (i.e, OBS) that could benefit from this. For context, here's an example of how Loupedeck does it: 
![2023-04-30_09-40-07](https://user-images.githubusercontent.com/6500836/235341708-8d0441a6-9395-4304-b253-9166bcc13b41.png)

Thanks!

**Comment by ChekTek** [comment]
Hi Martijn,

I will take this feedback to the team! In the meantime, we typically recommend reducing the number of actions in the list by combining them into settings in the property inspector. For example, the emoji actions could be combined into a single `Emoji` action, and then in the property inspector you could provide a dropdown for which emoji the user is looking to use on that key. 

https://developer.elgato.com/documentation/stream-deck/sdk/style-guide/#grouping

Best,
Zack

**Comment by GeekyEggo** [comment]
Hi @smitmartijn, quick update on this - the actions list within Stream Deck is something that's on the team's roadmap, and they're evaluating how it can evolve with Stream Deck as the application moves forward. Whilst we don't have any information to share on that at this stage, we do appreciate your feedback,

I'm going to close this discussion for now, but thank you again. 🙌
